e3sm_diags/driver/aerosol_budget_driver.py

```diff
@@ -1,114 +1,38 @@
+"""
```

Comment:
Most of the code in `aerosol_budget_driver.py` is similar to the CDAT version. I renamed and re-organized the functions, updated code to reflect Xarray based logic, and use xCDAT for regridding hybrid to pressure.